deploy-manage/deploy/cloud-enterprise/ece-install-offline-no-registry.md

```diff
@@ -19,16 +19,16 @@ To perform an offline installation without a private Docker registry, you have t
     docker pull docker.elastic.co/cloud-assets/kibana:7.17.27-0
     docker pull docker.elastic.co/cloud-assets/apm:7.17.27-0
     docker pull docker.elastic.co/cloud-assets/enterprise-search:7.17.27-0
-    docker pull docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.1
-    docker pull docker.elastic.co/cloud-release/kibana-cloud:8.17.1
-    docker pull docker.elastic.co/cloud-release/elastic-agent-cloud:8.17.1
-    docker pull docker.elastic.co/cloud-release/enterprise-search-cloud:8.17.1
+    docker pull docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.4
+    docker pull docker.elastic.co/cloud-release/kibana-cloud:8.17.4
+    docker pull docker.elastic.co/cloud-release/elastic-agent-cloud:8.17.4
+    docker pull docker.elastic.co/cloud-release/enterprise-search-cloud:8.17.4
     ```
 
     For example, for Elastic Cloud Enterprise 3.8.1 and the Elastic Stack versions it shipped with, you need:
 
     * Elastic Cloud Enterprise 3.8.1
```

Comment:
These documents mention 3.8 everywhere, could be the time to do a quick search-replace for 4.0